updatecli/weekly.d/jenkinscontroller-agents-packer-agent-all-in-one.yaml

```
@@ -0,0 +1,63 @@
+---
+name: Bump Packer Agent Templates (all-in-one) Version
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+sources:
+  packerImageVersion:
+    kind: githubrelease
+    spec:
+      owner: "jenkins-infra"
+      repository: "packer-images"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+
+sources:
+  getLatestInboundAllInOneContainerImageX86:
+    kind: dockerdigest
+    name: Get digest of the jenkinsciinfra/jenkins-agent-ubuntu-22.04 image
+    dependson:
+      - packerImageVersion
+    spec:
+      image: "jenkinsciinfra/jenkins-agent-ubuntu-22.04"
+      tag: '{{ source "packerImageVersion"}}'
+      architecture: amd64

```

```suggestion
sources:
  packerImageVersion:
    kind: githubrelease
    spec:
      owner: "jenkins-infra"
      repository: "packer-images"
      token: "{{ requiredEnv .github.token }}"
      username: "{{ .github.username }}"
  getLatestInboundAllInOneContainerImageX86:
    kind: dockerdigest
    name: Get digest of the jenkinsciinfra/jenkins-agent-ubuntu-22.04 image
    dependson:
      - packerImageVersion
    spec:
      image: "jenkinsciinfra/jenkins-agent-ubuntu-22.04"
      tag: '{{ source "packerImageVersion"}}'
      architecture: amd64

```


- If you open the YAML manifest in vscode (or any IDE supporting YAML schema validation), you should see a syntax error around "duplicated `sources` key".
- Executing the `updatecli diff` command sends the following error:

```
Loading Pipeline "updatecli/weekly.d/jenkinscontroller-agents-jnlp-container-jdk17.yaml"
ERROR: failed loading pipeline(s)
        * "updatecli/weekly.d/jenkinscontroller-agents-jnlp-container-jdk17.yaml" - yaml: unmarshal errors:
  line 25: mapping key "sources" already defined at line 16
```